Essentials/src/main/java/com/earth2me/essentials/commands/Commandrecipe.java

```diff
@@ -53,7 +53,23 @@ public void run(final Server server, final CommandSource sender, final String co
             throw new NotEnoughArgumentsException();
         }
 
-        final ItemStack itemType = ess.getItemDb().get(args[0]);
+        final ItemStack itemType;
+
+        if (args[0].equalsIgnoreCase("hand")) {
+            final User user = ess.getUser(sender.getPlayer());
+            if (user == null) {
+                throw new Exception(tl("recipeConsoleNoHands"));
```

Comment:
Is there a more generic message that could be used here? For example `This command cannot be run by console`.